velox/exec/tests/WindowFunctionRegistryTest.cpp

```diff
@@ -15,9 +15,13 @@
  */
 #include <gtest/gtest.h>
 
+#include "velox/common/base/tests/GTestUtils.h"
 #include "velox/exec/WindowFunction.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
```

Comment:
Is this include needed ? Also GTestUtils.h ?

Reply:
Fixed, thanks for pointing it out.